devOptions.html

```diff
@@ -1,238 +1,154 @@
-  
-  <button class="tabItem" onclick="openTab(event, 'CurrentLocation')">Current Location</button>
-  
-  <button class="tabItem" onclick="openTab(event, 'Energy')">Energy</button>
-
-  <button class="tabItem" onclick="openTab(event, 'Supplies')">Supplies</button>
-
-  <button class="tabItem" onclick="openTab(event, 'Credits')">Credits</button>
+<form>
+  <div class="menu">
+    <h1>
+      <b>Developer Game Configuration Options</b>
+    </h1>
+    <p>Please fill in all the fields for the state varibles, regular default values are intially set</p>
+    <hr>
 
-  <button class="tabItem" onclick="openTab(event, 'Wormholes')">Wormholes</button>
+    <label for="size"><b>Game Board Dimensions: </b></label>
+    <input type="text" name="Dim" id="Dimensions" value="128,128">
 
-  <button class="tabItem" onclick="openTab(event, 'PlayStyle')">Play Style</button>
+    <label for=loc><b>Starting Location: </b></label>
+    <input type="text" name="loc" id="Location" value="100,100">
 
-</div>
+    <label for="energy"><b>Energy: </b></label>
+    <input type="text" name="energy" id="Energy" value="1000">
 
+    <label "supplies"><b>Supplies: </b></label>
+    <input type="text" name="supplies" id="Supplies" value="100">
 
+    <label for="credits"><b>Credits: </b></label>
+    <input type="text" name="credits" id="Credits" value="1000">
 
+    <label for="wormhole"><b>Wormhole behavior: </b></label><br>
+      <input type="radio" name="worm" id="Random" checked> Random <br>
+      <input type="radio" name="worm" id="Fixed"> Fixed <br><br>
 
+    <label for=fixedworm><b>Fixed Wormhole Location: </b></label>
+    <input type="text" name="wormF" id="FixedWormhole" value="20,20">
 
-<!-- These are the contents of the tab blocks State Varibles-->
-<div id="GameSize" class="tabcontent">
-   <p>Please enter the width and height of the desired game board as Width, Height, for example 15, 20</p>
-        <label for=size>Game Board Dimensions:  </label>
-        <input type="text" name="Dim" id="Dimensions" value=""><br><br>
-</div>
+    <label for="play"><b>Play behavior: </b></label><br>
+        <input type="radio" name="playtype" id="Mortal" checked> Mortal <br>
+        <input type="radio" name="playtype" id="Immortal"> Immortal <br></br>
 
-<div id="CurrentLocation" class="tabcontent">
-  <p>Please enter the location you would like the ship to begin at as Xcoor, Ycoor, for example 20, 25 <br> However, keep in mind that this will be limited by the game size.</p>
-      <label for=loc>Starting Location:  </label>
-      <input type="text" name="loc" id="Location" value=""><br><br>  
-</div>
 
-<div id="Energy" class="tabcontent">
-  <p>Please enter the amount of energy you would like to begin with:</p>
-    <label for="energy">Energy: </label>    
-    <input type="text" name="energy" id="energy" value=""><br><br>
-</div>
+    <br><br>
+    <a type="submit" class="SubmitData" href="SpaceHunt.html" target="_self" onclick="setStateData();">Submit Data</a> 
```

Comment:
I don't think `class="SubmitData"` makes a lot of sense here since this button is the only thing with this class. How bout `id="devSubmitBtn"` or something like that instead?

Reply:
> I don't think `class="SubmitData"` makes a lot of sense here since this button is the only thing with this class. How bout `id="devSubmitBtn"` or something like that instead?

The class attribute is what applies the CSS style to the button, if its changed to id it changes the button to a very out of place looking link so I think class is fine. However, I did change the name to devSubmitBtn since it is more clear.

Reply:
Well you can select by id like this `#devSubmitBtn` in CSS

Reply:
I just figured that out as well, changes have been made however, I still can't get the link tag to work with the now external CSS file.

Reply:
Never mind, just solved it all. Completing the last requested change and pushing a commit to the branch.